### PR TITLE
fix: disappearing confirmation modal

### PR DESCRIPTION
**Pull request by joyce-shi**
## Notion ticket link

[Fix disappearing confirmation modal](https://www.notion.so/uwblueprintexecs/Fix-disappearing-confirmation-modal-a115b45dfc614591bcf0384c41b91f5b?pvs=4)

## Implementation description
* Instead of showing a successfully removed message, showing a Toast message to match the implementation for assessment table updates

## What should reviewers focus on?
* Works as described

### CODE REVIEW

File: frontend/src/components/user-management/RemoveUserModal.tsx [hunk 1]
```diff
@@ -39,41 +33,34 @@ const RemoveUserModal = ({
     },
   );
 
+  const { showToast } = Toast();
```
Comment (carissa-tang) [inline frontend/src/components/user-management/RemoveUserModal.tsx:36]: can we add a bit extra spacing between the top of the screen and the toast?

Comment (joyce-shi) [reply frontend/src/components/user-management/RemoveUserModal.tsx:36]: the toast component from chakra has preset position [options](https://chakra-ui.com/docs/components/toast/usage#changing-the-toast-position) and i don't think you can customize those. if you want to increase spacing, i think you have to build a custom toast component (i.e from `<Box>`). do you want me to do that?

Comment (carissa-tang) [reply frontend/src/components/user-management/RemoveUserModal.tsx:36]: does it work if you update the custom theme? smth like
```
const Alert = {
  baseStyle: {
    container: {
      ...style...
    }
  }
}
```

Comment (joyce-shi) [reply frontend/src/components/user-management/RemoveUserModal.tsx:36]: yes. LOL

File: frontend/src/themes/components/alert.ts [hunk 2]
```diff
@@ -40,13 +40,15 @@ const Alert = {
     },
     successToast: {
       container: {
+        margin: "4",
```
Comment (carissa-tang) [inline frontend/src/themes/components/alert.ts:43]: do we want this margin thing in `baseStyle` so it applies to all variants? not sure what the other two are tho / where it's used

Comment (joyce-shi) [reply frontend/src/themes/components/alert.ts:43]: toast uses alert under the hood, so i don't think so? bc we don't want to apply the margin to other non-toast alerts (i.e form error messages). let me know if i'm missing something b/c i am not on it today

Comment (carissa-tang) [reply frontend/src/themes/components/alert.ts:43]: ok ya that makes sense

**Review by carissa-tang — APPROVED** [review]

**Pull request merged**